deepmd/model/model.py

```diff
@@ -261,14 +267,31 @@ def build_descrpt(
                 suffix=suffix,
                 reuse=reuse,
             )
-            dout = tf.identity(dout, name="o_descriptor")
+            dout = tf.identity(dout, name="o_descriptor" + suffix)
```

Comment:
the suffix is to distinguish the different descriptors of dm dmmm models, am I correct?

Reply:
Yes. (qm and qmmm model)